src/main/java/seedu/address/logic/commands/CloseTutorialCommand.java

```diff
@@ -0,0 +1,106 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TUTORIAL;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.participation.Participation;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+import seedu.address.model.tutorial.Tutorial;
+
+/**
+ * Closes a tutorial in the system by unenrolling all students and marking the tutorial as closed.
+ */
+public class CloseTutorialCommand extends Command {
+
+    public static final String COMMAND_WORD = "closetut";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Closes the tutorial identified by the subject used in the displayed tutorial list.\n"
+            + "Parameters: "
+            + PREFIX_TUTORIAL + "SUBJECT_NAME\n"
+            + "Example: " + COMMAND_WORD
+            + PREFIX_TUTORIAL + "Chemistry";
+
+    public static final String MESSAGE_CLOSE_TUTORIAL_SUCCESS = "Successfully closed tutorial: %1$s class";
+    public static final String MESSAGE_TUTORIAL_NOT_FOUND = "No tutorial class with the name %1$s is found.";
```

Comment:
Maybe we could shift this to Messages class so i could also use it for marking attendance

Reply:
Ahhh okay sure i will shift it.